lms/static/styles/components/_Dialog.scss

```diff
@@ -40,26 +40,15 @@
 .Dialog__title {
   display: flex;
   align-items: center;
+  line-height: 1em;
   font-size: var.$title-font-size;
 }
 
-.Dialog__cancel-btn {
-  @include focus.outline-on-keyboard-focus;
-  border: 0;
-  background: none;
-  color: var.$grey-6;
-
-  // Given the button a large hit target and ensure the 'X' label is large
-  // enough to see easily and aligned with the right edge of the dialog.
-  // Add negative margins so that the button does not force the dialog to
-  // grow in height.
-  font-size: var.$title-font-size;
-  padding: 5px;
-  margin: -10px 0px;
-
-  &:hover {
-    cursor: pointer;
-    color: var.$brand;
+.Dialog__cancel-icon-btn {
```

Comment:
Please isolate/consolidate all customizations of the shared button styling into a single module, `buttons.scss`. See the client for how this is done there.

Reply:
Actually, wait. Can you walk me through what this customization is trying to achieve?

Reply:
Thanks! that seems reasonable. In the future, I'll make a note to add the buttons.scss file which we will probably need to get into with the grader.